src/engine/client/cl_cgame.cpp

```diff
@@ -604,11 +604,7 @@ static int LAN_ServerIsVisible( int source, int n )
 				return cls.localServers[ n ].visible;
 			}
 
-		if ( Cmd_Argc() == 1 )
-		{
-			Log::Warn("Server sent a pubkey_decrypt command, but sent nothing to decrypt!" );
-			return false;
-		}
+			break;
```

Comment:
what's that?

Reply:
Yeah it's weird that this cade was there.

Reply:
That code was wrong. Probably the result of a bad merge conflict resolution. You can find the exact same code earlier in the file.

Reply:
okie!